lib/app/features/users/data/repositories/users_repository.dart

```diff
@@ -82,4 +113,12 @@ extension _FutureExtension<T extends String> on Future<T> {
     return then((v) => jsonDecode(v) as Map<String, dynamic>)
         .then((v) => UserSearchSessionModel.fromJson(v));
   }
+
+  Future<ValidField> parseBlockOrReport() async {
```

Comment:
Talvez ficaria mais semântico colocar um nome que remeta ao objeto de retorno, o que acha?
```suggestion
  Future<ValidField> parseValidField() async {
```